runtime/compiler/runtime/Runtime.cpp

```diff
@@ -167,6 +168,29 @@ void* TR_RuntimeHelperTable::getFunctionPointer(TR_RuntimeHelper h)
 
 void* TR_RuntimeHelperTable::getFunctionEntryPointOrConst(TR_RuntimeHelper h)
    {
+#if defined(J9VM_OPT_JITSERVER) && defined(TR_HOST_POWER)
+   TR::CompilationInfo *compInfo = getCompilationInfo(jitConfig);
+   auto compInfoPT = TR::compInfoPT;
+   if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER &&
+       compInfoPT)
+      {
+      // Return the client-side helper address
+      // Get cached value if it exists, otherwise make remote and cache
+      void *helperAddress = compInfoPT->getClientData()->getRuntimeHelper(h);
+      if (helperAddress)
+         return helperAddress;
+      auto stream = TR::CompilationInfo::getStream();
```

Comment:
The stream should be available from CompInfoPT->getMethodToBeCompiled()->_stream or something like that